tests/federation/test_printer.py

```diff
@@ -346,7 +346,7 @@ def top_products(self, first: int) -> List[Product]:
     schema = strawberry.federation.Schema(query=Query)
 
     expected = """
-        extend type Product @key(fields: "upc", resolvable: "True") {
+        extend type Product @key(fields: "upc", resolvable: true) {
```

Comment:
This expected value is incorrect, the type is a bool, and bool is a valid type for a directive argument. https://spec.graphql.org/October2021/#sec-Language.Directives